tools/flaky_test_monitor/common/level2.go

```diff
@@ -21,10 +22,17 @@ type Level2TestResult struct {
 	Passed          int       `json:"passed"`
 	Failed          int       `json:"failed"`
 	Skipped         int       `json:"skipped"`
-	NoResult        int       `json:"no_result"`
-	FailureRate     float32   `json:"failure_rate"`
-	AverageDuration float32   `json:"average_duration"`
-	Durations       []float32 `json:"durations"`
+	Exceptions      int       `json:"exceptions"`
+	FailureRate     float64   `json:"failure_rate"`
+	AverageDuration float64   `json:"average_duration"`
+	Durations       []float64 `json:"durations"`
+}
+
+func checkFloatEqual(t *testing.T, expected, actual float64, message string) {
```

Comment:
this looks like a utility function that doesn't belong here. Maybe into `utils/unittest.go` or `tools/flaky_test_monitor/common/utility.go`